terraform/environments/sprinkler/secrets.tf

```diff
@@ -1,13 +1,16 @@
-# Get secret by arn for environment management
-data "aws_ssm_parameter" "environment_management_arn" {
-  name = "environment_management_arn"
+# Get modernisation account id from ssm parameter
```

Comment:
I think these can be removed from this file as they are covered in the platform_secrets file?  I think this is left over from where we used to have a single secrets file, but it was a bit confusing to users so we split it out into secrets.tf for users to put their application secrets and platform_secrets.tf where the required secrets for running the Terraform are located.

Reply:
I'll rename the file to platform_secrets.tf